Auth-MSSC/src/main/java/com/clinics/auth/ui/repositorie/UserRepository.java

```diff
@@ -11,6 +12,8 @@
 public interface UserRepository extends JpaRepository<User, Long> {
```

Comment:
There's a typo in package name - should be repository, and why is this altogether with controller, service and model inside a ui package ?

Reply:
Done, into ui only have controlers, services, and clients, data will have only bootstrap, model, repository